DiscountDiscover/View Controllers/ProfileViewController.m

```diff
@@ -13,10 +13,32 @@
 
 @interface ProfileViewController ()
 
+@property (weak, nonatomic) IBOutlet UIImageView *profileView;
+@property (weak, nonatomic) IBOutlet UILabel *nameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *emailLabel;
+@property (weak, nonatomic) IBOutlet UILabel *preferencesLabel;
+
 @end
 
 @implementation ProfileViewController
 
+- (void)viewDidLoad {
+    [super viewDidLoad];
+   
+    PFUser *user = [PFUser currentUser];
+    self.nameLabel.text = user[@"name"];
```

Comment:
You should use `user.name` instead of `user[@"name"]` so the compiler can catch any issues. Same for all the other user attributes below. If Parse doesn't support this you should build a `User` class to wrap `PFUser` and do the dictionary parsing there and expose regular properties.